spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/cloud_manager_spec.rb

```diff
@@ -0,0 +1,55 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager do
+  context ".raw_connect" do
+    let(:config) do
+      {
+        :provider                 => "IbmCloud",
+        :IbmCloud_project         => "project",
+        :IbmCloud_json_key_string => "encrypted",
+        :app_name                 => Vmdb::Appliance.PRODUCT_NAME,
+        :app_version              => Vmdb::Appliance.VERSION,
+      }
+    end
+  end
```

Comment:
My comment got marked outdated but it still applies, we should either add tests to this context or remove it